common/reviews/api/node-core-library.api.md

```diff
@@ -791,6 +797,13 @@ export enum NewlineKind {
     OsDefault = "os"
 }
 
+declare namespace Objects {
```

Comment:
```suggestion
declare namespace Object {
```

Reply:
We also have `MapExtensions` in NCL... we should probably commonize on a naming scheme here.

Reply:
`Object` is an existing ECMAScript global. Very bad things happen if you collide with the name.